### DISCUSSION TITLE
Active Learning using transformers

### DISCUSSION THREAD

**Discussion by vahuja4** [cleanlab/cleanlab#874] [Q&A]
Going through this notebook: https://github.com/cleanlab/examples/blob/master/active_learning_transformers/active_learning.ipynb

Is there any reason why the model is retrained from scratch in the active learning loop as opposed to training based on the previous checkpoint?

**Comment by tataganesh** [comment]
Hi @vahuja4. Could you please indicate which part of the code are you referring to?
